docker/docker-compose.yml

```diff
@@ -5,12 +5,24 @@ services:
     environment:
       - ZOOKEEPER_CLIENT_PORT=32181
       - ZOOKEEPER_SERVER_ID=1
+  minio:
+    image: minio/minio
+    command: server /data
+    volumes:
+      - '/mnt/data:/data'
+      - '/mnt/config:/root/.minio'
```

Comment:
Getting these mounts to work requires disabling SELinux as the host mount points do not have the proper security context. We should probably come up with a way to set the directories to use the proper context prior to starting up.

Not sure if we can do this within docker-compose, but we can at least make a note in the README about it. 
`chcon -R -u system_u -t container_file_t /mnt/data`
`chcon -R -u system_u -t container_file_t /mnt/config`

Reply:
It seems to work as expected with SELinux with `:Z`

```yaml
      - /mnt/data:/data:Z
      - /mnt/config:/root/.minio:Z
```